netlify.toml

```diff
@@ -4,3 +4,8 @@ command = "npm run build:preview"
 
 [context.production]
 command = "npm run build:production"
+
+[[redirects]]
+from = "https://docs.opentelemetry.io/*"
+to = "https://opentelemetry.io/docs/:splat"
```

Comment:
Hmm, I wondering if we can just use a path here so that Netlify redirects to the default domain? I.e., just use `/docs/:splat`.